.semaphore/semaphore.yml

```
@@ -0,0 +1,17 @@
+version: v1.0
+name: Initial Pipeline
+agent:
+  machine:
+    type: e1-standard-2
+    os_image: ubuntu1804
+blocks:
```

There is not a lot of work done on Railjet, but the auto-cancel feature could be useful to add as default..
```suggestion
auto_cancel:
  running:
    when: "branch != 'master'"
blocks:
```